pkg/controller/container-runtime-config/helpers.go

```diff
@@ -538,7 +540,7 @@ func getValidBlockedAndAllowedRegistries(releaseImage string, imgSpec *apicfgv1.
 		// if there is a match, return all the blocked registries except those that matched and return an error as well
 		if runtimeutils.ScopeIsNestedInsideScope(payloadRepo, reg) {
 			// If the payload registry doesn't have mirror rules configured for it, then don't add it to the blocked registries list
```

Comment:
```suggestion
			// If the payload registry doesn't have mirror rules configured for it, then don't add it to the blocked registries list
			// Note that we care only about digest mirrors (and not ImageTagMirrorSet) because the OpenShift release payload only uses digest references.
```

Reply:
comment fixed.